### PR TITLE
Update S3 bucket path

### PR DESCRIPTION
**Pull request by jharenza**
- Update data bucket due to account switching

#### What scientific question is your analysis addressing?
update OpenPedCan bucket path due to a need to switch accounts

#### What was your approach?
update OpenPedCan bucket path in:
- `download-data.sh`

#### What GitHub issue does your pull request address?
https://github.com/PediatricOpenTargets/ticket-tracker/issues/184

#### Which areas should receive a particularly close look?
Check the download

#### Is there anything that you want to discuss further?
No

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
No

### Checklist
- [x] Is download-data.sh up to date?
- [x] Was download-data.sh tested and did it complete without error?

### CODE REVIEW

**Review by runjin326 — APPROVED** [review]
Thanks for sharing this, @jharenza! 
I also tried running the download script - works for me!

**Comment by jharenza** [comment]
thanks for the review @runjin326 - just want to confirm you have write access to this bucket before we merge

**Comment by runjin326** [comment]
@jharenza, yes uploaded one file and deleted - worked fine for me!

**Pull request merged**
